src/Resources/app/administration/src/module/sw-in-app-purchases/types/index.ts

```diff
@@ -1,18 +1,22 @@
 export type { Extension } from 'src/module/sw-extension/service/extension-store-action.service';
 
-export type InAppPurchasePriceModel<Type extends 'rent' | 'buy' = 'rent' | 'buy'> = {
-    type: Type;
+export type InAppPurchasePriceModel = {
+    type: 'rent' | 'buy';
     price: number;
-    duration: Type extends 'rent' ? number : undefined | null;
-    oneTimeOnly: Type extends 'buy' ? boolean : undefined | null;
+    duration: number | null;
+    oneTimeOnly: null;
+    variant: string;
+    conditionsType?: string | null;
```

Comment:
the old type still holds up, `duration` is still only set if type is `rent` and so on